shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-mysql/src/main/java/org/apache/shardingsphere/proxy/frontend/mysql/MySQLErrPacketFactory.java

```diff
@@ -47,7 +47,7 @@ public static MySQLErrPacket newInstance(final int sequenceId, final Exception c
         if (cause instanceof SQLException) {
             SQLException sqlException = (SQLException) cause;
             return null != sqlException.getSQLState() ? new MySQLErrPacket(sequenceId, sqlException.getErrorCode(), sqlException.getSQLState(), sqlException.getMessage())
-                : new MySQLErrPacket(sequenceId, MySQLServerErrorCode.ER_INTERNAL_ERROR, sqlException.getCause().getMessage());
+                : new MySQLErrPacket(sequenceId, MySQLServerErrorCode.ER_INTERNAL_ERROR, cause.getMessage());
```

Comment:
When `null == sqlException.getSQLState() `, `null == sqlException.getCause()` but `cause.getMessage()` could run well, right?

Reply:
lready add the test code

Reply:
Sorry, I did not catch you. 
Which condition will make `sqlException.getCause isNull` ?

Reply:
![image](https://user-images.githubusercontent.com/664050/89249724-8a52ef80-d645-11ea-8e6d-229ed00790fc.png)

Reply:
@cmonkey Sorry?
Or what's the difference between `sqlException.getCause().getMessage()` and `cause.getMessage()`?

Reply:
val exception = new SQLException()
MySQLErrPacketFactory.newInstance(1, exception)
cause.isInstanceOf[SQLException) is true 

cause.asInstanceOF[SQLException) to  sqlException
sqlException.SQLState is null 
sqlException.cuase is not null 
sqlException.getCuase is null

Reply:
Hi @cmonkey Thanks for your explanation. Make sense to me.
I will merge this PR after CI passes. Could you give `travis-ci` a look? I found it failed. Thanks a lot.